gqlalchemy/exceptions.py

```diff
@@ -119,3 +127,15 @@ class GQLAlchemyExtraKeywordArgumentsInWhere(GQLAlchemyError):
     def __init__(self):
         super().__init__()
         self.message = EXTRA_KEYWORD_ARGUMENTS_IN_WHERE
+
+
+class GQLAlchemyMissingAliasInReturn(GQLAlchemyError):
+    def __init__(self):
+        super().__init__()
```

Comment:
remove parameter-less base constructor invocation

Reply:
Did you mean that I should delete line 134 - `super().__init__()`?
Should I then delete it on other constructors as well?